src/stylesheets/elements/_buttons.scss

```diff
@@ -1,181 +1,215 @@
+  @include u-border("base-light");
 
   &.usa-button--hover,
   &:hover {
-    color: color('gold-70');
-    background-color: color('white');
-    box-shadow: $button-stroke color('base');
+    color: color("gold-70");
+    background-color: color("white");
+    //box-shadow: $button-stroke color("base");
+    @include u-border("base");
   }
 
   &.usa-button--active,
   &:active {
-    color: color('white');
-    background-color: color('base-dark');
+    color: color("white");
+    background-color: color("base-dark");
+    @include u-border("base-dark");
   }
 }
 
 .usa-button--secondary {
-  background-color: color('primary-lighter');
-  box-shadow: $button-stroke color('primary');
+  background-color: color("primary-lighter");
+  // box-shadow: $button-stroke color("primary");
 
   &.usa-button--hover,
   &:hover {
-    box-shadow: $button-stroke color('primary-light');
-    background-color: color('white');
+    // box-shadow: $button-stroke color("primary-light");
+    background-color: color("white");
   }
 
   &.usa-button--active,
   &:active {
-    color: color('secondary-darker');
-    box-shadow: $button-stroke color('secondary-darker');
-    background-color: color('white');
+    color: color("secondary-darker");
+    // box-shadow: $button-stroke color("secondary-darker");
+    background-color: color("white");
   }
 
   &.usa-button--disabled,
   &:disabled {
-    color: color('disabled');
-    box-shadow: $button-stroke color('disabled');
+    color: color("disabled");
+    // box-shadow: $button-stroke color("disabled");
+    @include u-border("disabled");
     background-color: transparent;
   }
 }
 
 .usa-button--danger {
-  background-color: color('error');
+  background-color: color("error");
+  @include u-border("error");
 
   &.usa-button--hover,
   &:hover {
-    background-color: color('error-lighter');
+    background-color: color("error-lighter");
+    @include u-border("error-lighter");
   }
 
   &.usa-button--active,
   &:active {
-    color: color('white');
-    background-color: color('error-dark');
+    color: color("white");
+    background-color: color("error-dark");
+    @include u-border("error-dark");
   }
 }
 
-// TODO: Remove this style
-// as its only used for example display
-.button-label {
-  text-align: center;
-  display: inline-block;
-  margin: 10px;
 
-  label {
-    display: block;
-    font-size: 14px;
+.usa-button--white {
+  background-color: color("white");
+  @include u-border("white");
+
+  &.usa-button--hover,
+  &:hover {
+    background-color: color("base-lighter");
+    @include u-border("base-lighter");
   }
-  &--right{
-    text-align: right;
 
+  &.usa-button--active,
+  &:active {
+    color: color("white");
+    background-color: color("base-light");
+    @include u-border("base-light");
   }
-  &--left{
-    text-align: left;
+}
+.usa-button--accent {
+  background-color: color("accent-cool");
+  @include u-border("accent-cool");
 
+  &.usa-button--hover,
```

Comment:
USWDS also has a .usa-button--hover class for displaying so may want to make this .sds-button--hover so we don't inherit properties such as border-bottom: 0;